store/store_test.go

```diff
@@ -36,7 +37,9 @@ func testBasic(t *testing.T, s Store) {
 	v, err = s.Get(TK2)
 	a.NoError(err)
 	a.Equal(TV2, v)
-	a.Equal([]Key{TK1, TK2}, s.Keys())
```

Comment:
I think I recall expecting that test to intermittently fail due to random order, but didn't see any test failures on a few re-runs so didn't pre-emptively fix.

Ah...test cacheing. That'll be it.

Thank you!